documentation/modules/exploit/linux/http/vinchin_backup_recovery_cmd_inject.md

```diff
@@ -0,0 +1,101 @@
+# Vinchin Backup & Recovery Command Injection Exploit Documentation
```

Comment:
The formatting of this documentation file looks slightly different than what we might expect. We have a guide template here you can reference: https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/module_doc_template.md.
Once you've made some changes you can run: 
```
 ruby tools/dev/msftidy_docs.rb documentation/modules/exploit/linux/http/vinchin_backup_recovery_cmd_inject.md
 ```
 The above script will highlight any remaining formatting error that might exist in the doc file.